src/herder/test/HerderTests.cpp

```diff
@@ -2564,11 +2564,6 @@ TEST_CASE("SCP State", "[herder]")
         };
 
     auto doTest = [&](bool forceSCP) {
-        SECTION("sqlite")
```

Comment:
Why did you remove this section? It deals with persisting SCP stuff, not ledger state. We still support SQLite for misc data like consensus

Reply:
`TESTDB_BUCKET_DB_PERSISTENT` is the best, most accurate to prod testing mode we have. It uses BucketListDB for ledger state queries, but still maintains all the same sqlite tables that prod does, so this test mode is still hitting all the misc sqlite tables. This in conjunction with the postgres section is complete coverage.

Reply:
Oh I see, and `postgres` is left to test scp persistence on postgres, even though it's technically a deprecated backend for ledger state.